app/components/channel_list/categories/categories.tsx

```diff
@@ -44,6 +48,13 @@ const Categories = (props: Props) => {
             style={styles.flex}
             showsHorizontalScrollIndicator={false}
             showsVerticalScrollIndicator={false}
+            keyExtractor={extractKey}
+            removeClippedSubviews={true}
+            initialNumToRender={5}
+            windowSize={15}
+            updateCellsBatchingPeriod={10}
+            maxToRenderPerBatch={5}
```

Comment:
why make it smaller than the default? what is the gain here?

Reply:
Broadly;
1. Render a lower count of items, but run the render more often
2. Render a higher count of items, but run the less often
 

Both have tradeoffs, but option 2 means the user *will* see an empty list while the batching catches up to their scroll position. Option 1 uses more processing power (offset by lower amounts to render), but means the UI is updating more often and is more likely to keep up with a users scroll.

https://reactnative.dev/docs/optimizing-flatlist-configuration#updatecellsbatchingperiod

Reply:
The idea is that we give this values a try and then if needed we can start adjusting if it feels that is rendering too much or scrolling generate blank areas.